package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lenne.tech/cli",
-  "version": "0.0.17",
+  "version": "0.0.18",
```

Comment:
If you change the version number in package.json, please also change the version number in package-lock.json.